lib/ssz_ex.ex

```diff
@@ -331,39 +331,25 @@ defmodule LambdaEthereumConsensus.SszEx do
     root |> hash_nodes(serialized_len)
   end
 
+  # Currently, we are not using this one.
```

Comment:
Let's make it a todo:
```suggestion
  # TODO: we are not using this
```